receiver/jaegerreceiver/trace_receiver_test.go

```diff
@@ -137,6 +140,62 @@ func TestGRPCReception(t *testing.T) {
 
 }
 
+func TestGRPCReceptionWithTLS(t *testing.T) {
+	// prepare
+	grpcServerOptions := []grpc.ServerOption{}
+	tlsCreds := receiver.TLSCredentials{
+		CertFile: path.Join(".", "testdata", "certificate.pem"),
+		KeyFile:  path.Join(".", "testdata", "key.pem"),
+	}
+
+	tlsOption, _ := tlsCreds.ToGrpcServerOption()
+
+	grpcServerOptions = append(grpcServerOptions, tlsOption)
+
+	config := &Configuration{
+		CollectorGRPCPort:    14250, // that's the only one used by this test
+		CollectorGRPCOptions: grpcServerOptions,
+	}
+	sink := new(exportertest.SinkTraceExporter)
+
+	jr, err := New(context.Background(), config, sink)
+	assert.NoError(t, err, "should not have failed to create a new receiver")
+	defer jr.StopTraceReception()
+
+	mh := receivertest.NewMockHost()
+	err = jr.StartTraceReception(mh)
+	assert.NoError(t, err, "should not have failed to start trace reception")
+	t.Log("StartTraceReception")
+
+	creds, err := credentials.NewClientTLSFromFile(path.Join(".", "testdata", "certificate.pem"), "opentelemetry.io")
+	require.NoError(t, err)
+	conn, err := grpc.Dial(fmt.Sprintf("0.0.0.0:%d", config.CollectorGRPCPort), grpc.WithTransportCredentials(creds))
+	require.NoError(t, err)
+	defer conn.Close()
+
+	cl := api_v2.NewCollectorServiceClient(conn)
+
+	now := time.Unix(1542158650, 536343000).UTC()
```

Comment:
What are the magic numbers? Why is this called `now`? Should this be a fixed timestamp or `time.Now()` can be actually used?

Reply:
time.now should be fine